src/improvements/tasks/MultisigTask.sol

```diff
@@ -463,51 +521,22 @@ abstract contract MultisigTask is Test, Script, StateOverrideManager, TaskManage
         );
     }
 
-    /// @notice Helper function to get the approve transaction info.
-    function getApproveTransactionInfo(Action[] memory actions)
-        internal
-        view
-        returns (bytes memory callData, bytes memory encodedTxData, bytes32 domainSeparator, bytes32 messageHash)
-    {
-        callData = generateApproveMulticallData(actions);
-        encodedTxData = getEncodedTransactionData(childMultisig, callData);
-        (domainSeparator, messageHash) =
-            GnosisSafeHashes.getDomainAndMessageHashFromEncodedTransactionData(encodedTxData);
-        return (callData, encodedTxData, domainSeparator, messageHash);
-    }
-
     /// @notice Returns true if the given account access should be recorded as an action. This function is used to filter out
     /// actions that we defined in the `_build` function of our template. The actions selected by this function will get executed
     /// by the relevant Multicall3 contract (e.g. `Multicall3` or `Multicall3DelegateCall`).
-    function _isValidAction(VmSafe.AccountAccess memory access, uint256 topLevelDepth) internal view returns (bool) {
+    function _isValidAction(VmSafe.AccountAccess memory access, uint256 topLevelDepth, address rootSafe)
+        internal
+        view
+        returns (bool)
+    {
         bool accountNotRegistryOrVm =
             (access.account != AddressRegistry.unwrap(addrRegistry) && access.account != address(vm));
         bool accessorNotRegistry = access.accessor != AddressRegistry.unwrap(addrRegistry);
         bool isCall = (access.kind == VmSafe.AccountAccessKind.Call && access.depth == topLevelDepth);
         bool isTopLevelDelegateCall =
             (access.kind == VmSafe.AccountAccessKind.DelegateCall && access.depth == topLevelDepth);
-        bool accessorIsParent = (access.accessor == parentMultisig);
-        return accountNotRegistryOrVm && accessorNotRegistry && (isCall || isTopLevelDelegateCall) && accessorIsParent;
-    }
-
-    /// @notice Composes a description string for the given access using the provided operation string.
-    function _composeDescription(VmSafe.AccountAccess memory access, string memory opStr)
```

Comment:
This function has been deleted. Not need for it to be a separate function. In a later PR this MultisigTask.sol file is refactored further.